sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/cli/OwaspZapScanExecutor.java

```diff
@@ -39,15 +38,29 @@ public void execute(OwaspZapScanContext scanContext) throws ZapWrapperRuntimeExc
 
     }
 
+    private void assertTargetReachable(OwaspZapScanContext scanContext) {
+        boolean isReachable = false;
+        for (String url : scanContext.getOwaspZapURLsIncludeList()) {
```

Comment:
Did I understand this, you are iterating over all URLs from the include list? Meaning if I have 500 urls in the list it would iterate until at least one URL is reachable?

Reply:
We shortly discussed this issue and thought of this solution. Of course in case of many includes, that are all not reachable, the connection check can take some time.